manifests/charts/istio-control/istio-discovery/templates/clusterrole.yaml

```diff
@@ -0,0 +1,163 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: istiod{{- if not (eq .Values.revision "")}}-{{ .Values.revision }}{{- end }}
```

Comment:
This can conflict if we have other istiod's in other namespaces. We may need namespace and revision in the name?

Reply:
Makes sense, added `{{ .Release.Namespace }}` to the ClusterRole/Binding

Reply:
Actually, this reintroduces conflict with the original ClusterRole name. Don't want to add random suffixes to the resources but we somehow need to make them unique from old base. Maybe we make it `istiod-cluster-role-...`, bad style but harmless